app/views/users/vimeo_search.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="col-sm-6">
+  <% if @vimeo_results == nil %>
```

Comment:
It's a little weird to have an if statement without anything to execute. Since you're executing some code if @vimeo_results are present, then I'd recommend writing it like this:
```ruby
<% if @vimeo_results %>
  <h1>Search Results</h1>
  ...
<% end %>